installation/language/de-DE/joomla.ini

```diff
@@ -4,19 +4,33 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
+; Fatal error page
+; These will be processed by the JavaScript Build
+BUILD_FATAL_HEADER="Sorry, there was a problem we could not recover from."
```

Comment:
good idea for a translation? @zero-24 @SniperSister @Hackwar

Reply:
```suggestion
BUILD_FATAL_HEADER="Es ist ein Problem aufgetreten"
```